scripts/build.py

```diff
@@ -161,7 +163,11 @@ def generate_app_yaml(is_prod_mode=False):
             content = content.replace(
                 file_path,
                 prod_file_prefix + file_path)
-    if not is_prod_mode:
+    # The version: default line is required to run jobs on a local server (
+    # both in prod & non-prod mode). This line is not required when app.yaml
+    # is generated during deployment. So, we remove this if the build process
+    # is being run from the deploy script.
+    if is_deploy_mode:
```

Comment:
Earlier, `version: default` was not required in dev_mode, right? is it required now?

Reply:
Yes, it was required in dev mode as well. `app.yaml` is not generated for dev_mode and app_dev.yaml is directly used, so replacing it did not have any impact in dev mode.